examples/rest_api/mlflow_tracking.py

```diff
@@ -0,0 +1,71 @@
+"""
```

Comment:
Can you name the file `mlflow_tracking_rest_api.py` or `tracking_rest_api.py` instead to make it clear that this is an example of the REST API specifically?